estate/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
We try to always have the return at the end of the last line of every file.
This way, when new lines are added later, the formerly last line does not show up as being impacted in the diff.